windows/src/developer/TIKE/Tike.dpr

```diff
@@ -301,8 +303,8 @@ begin
       if TikeActive then Exit;
       InitClasses;
       Application.CreateForm(TmodWebHttpServer, modWebHttpServer);
-      Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
-      ShowStartup;
+  Application.CreateForm(TfrmKeymanDeveloper, frmKeymanDeveloper);
```

Comment:
Was the indentation supposed to disappear in this file? Does it even matter?

Reply:
Delphi automatically updates that file and flubs the indentation when it does. It doesn't matter; if I was a complete purist I'd go back and tidy it up again 😁